app/config.py

```diff
@@ -17,3 +17,7 @@
 MAGIC_LINK_TEMPLATE_ID = os.environ.get(
     "GOV_NOTIFY_TEMPLATE_KEY"
 ) or os.environ.get("TEST_MAGIC_LINK_TEMPLATE_ID")
+
+APPLICATION_RECORD_TEMPLATE_ID = os.environ.get(
+    "GOV_NOTIFY_APPLICATION_RECORD_TEMPLATE_KEY"
+) or os.environ.get("APPLICATION_RECORD_TEMPLATE_ID")
```

Comment:
What's the reasoning behind the two possible keys here?

Reply:
Local key for development since it needs to communicate with govuk-notify-service

Reply:
Since this in an environment variable we can set a different value locally. In that case I don't see the need for two keys.

Reply:
I agree, It's the same key with different names, one of them is stored locally for just testing purposes for now & when  I have it approved, I will remove one of the keys. 
We would only require GOV_NOTIFY_APPLICATION_RECORD_TEMPLATE_KEY but it is not stored on cf yet.